castle/cms/interfaces/controlpanel.py

```diff
@@ -153,6 +143,26 @@ class ISecuritySchema(controlpanel.ISecuritySchema):
         )
     )
 
+    public_url = schema.TextLine(
+        title=u'Public site URL',
+        description=u'The URL the public will use to view your site.',
+        default=None,
+        required=False)
+
+    backend_url = schema.Tuple(
+        title=u'Backend site URLs',
+        description=u'The URL(s) from which you will be editing and maintaining site content. '
+                    u'One per URL per line. Main URL first.',
```

Comment:
"One URL per line" (extra "per" in there). "Please enter the main URL first"  might be clearer too.